go.mod

```diff
@@ -164,6 +170,9 @@ replace (
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	// TODO tag after https://github.com/cosmos/cosmos-sdk/pull/14207 merge
```

Comment:
Small nit, these replace are in the wrong group.

Reply:
fixed